com.unity.render-pipelines.high-definition/Editor/Material/Decal/DecalProjectorEditor.cs

```diff
@@ -57,7 +57,7 @@ static void UpdateColorsInHandlesIfRequired()
         SerializedProperty m_FadeFactor;
         SerializedProperty m_DecalLayerMask;
 
-        int layerMask => (target as Component).gameObject.layer;
+        int layerMask => (target as DecalProjector).cachedEditorLayer;
```

Comment:
Should you also fix the below layerMaskHasMultipleValue for coherency then?

Reply:
Added the fix